nengo_ocl/tests/conftest.py

```diff
@@ -7,8 +7,9 @@
 from nengo.conftest import plt, rng  # noqa: F401
 
 
-ctx = cl.create_some_context()
```

Comment:
So the reason I had it this way is so that the same context would be used for all the tests. Recreating the context had led to some strange errors in the past (with running out of memory on the device if I recall correctly). If we want to have `ctx` be a fixture, I'd have a global `_ctx = cl.create_some_context()` here, and then have the fixture return `_ctx`.